app/src/main/java/net/sourceforge/opencamera/ui/RecSyncView.java

```diff
@@ -119,4 +109,18 @@ private Button addButton(@StringRes int text) {
 
         return button;
     }
+    static class Utils {
+
+        public static void alignCheckbox(CheckBox check_box, float scale) {
+            // align the checkbox a bit better
+            LayoutParams params = new LayoutParams(
+                    LayoutParams.MATCH_PARENT,
+                    LayoutParams.MATCH_PARENT
+            );
+            final int left_padding = (int) (10 * scale + 0.5f); // convert dps to pixels
```

Comment:
Maybe we have some misunderstanding, but I think that "(10 * scale + 0.5f)" is a conversion that should be in a separate class named UIUtils. At the same time align checkbox is used only here and doesn't need to be excluded into method.

Reply:
But this conversion is also used only here.
Actuaaly, a lot of code in this class is also used in `PopupView` (and is taken from there), but it's in an original OpenCamera file, so we don't want to change it to avoid any possible conflicts

Reply:
It is used for right_padding too, and moreover nearly all such custom views will require similar conversion

Reply:
Yes, original files should be unchanged